blocks/form/form.js

```diff
@@ -202,6 +207,9 @@ const createSelect = withFieldWrapper((fd) => {
     const option = document.createElement('option');
     option.textContent = o.trim();
     option.value = o.trim();
+    if (fd.Value === o.trim()) {
+      option.selected = true;
```

Comment:
option.selected = fd?.Value === o?.trim()

Reply:
I am not convinced with optional chaining since it doesn't work in older browsers

Reply:
94% coverage is there for optional https://caniuse.com/?search=optional%20chaining%20